tests/util/ConverterTestCase.py

```diff
@@ -10,9 +11,27 @@ class ConverterTestCase(usdex.test.TestCase):
 
     defaultUpAxis = UsdGeom.Tokens.z  # noqa: N815
 
+    defaultValidationIssuePredicates: typing.ClassVar[list[omni.asset_validator.IssuePredicates]] = [  # noqa: N815
+        # URDF USD Converter uses nested bodies, as it more faithfully matches the kinematic tree in URDF.
```

Comment:
Why did you decide to use nested bodies? I think the more faithful representation of the source data (albeit not the kinematic tree) would be a flat list of bodies & a flat list of joints.

@rgasoto for thoughts.

Reply:
We did discuss this yesterday. The feeling so far was that the MJC converter already places this requirement so we're less worried about having the same restriction here.